examples/dreambooth/train_dreambooth.py

```diff
@@ -768,7 +802,8 @@ def main(args):
                 encoder_hidden_states = text_encoder(batch["input_ids"])[0]
 
                 # Predict the noise residual
-                model_pred = unet(noisy_latents, timesteps, encoder_hidden_states).sample
+                with torch.cuda.amp.autocast(enabled=True):
```

Comment:
When we pass `--mixed_precision="fp16"` accelerate automatically uses `autocast`; we should not hardcode this here.